### PR TITLE
Fix: test run attachments

### PR DESCRIPTION
**Pull request by danymill**
issue: https://github.com/qaprosoft/zafira/issues/1993

### CODE REVIEW

File: client/app/containers/test-details/test-details.html [hunk 1]
```diff
@@ -137,17 +137,26 @@
                                             <p name="testName" class="test-info__name" data-ng-class="test.status === 'QUEUED' ? 'grey_font' : 'black_font'">{{::test.name}}
                                                 <span data-ng-if="test.blocker" class="badge ng-binding bug-label-bg">BLOCKER</span></p>
                                             <div class="test-info__wrapper">
-                                                <span data-ng-if="test.finishTime && (test.finishTime - test.startTime)/1000 > 0 && test.status !== 'QUEUED'" data-ng-class="test.status === 'QUEUED' ? 'grey_font' : 'light_text'">
+                                                <span ng-if="test.finishTime && (test.finishTime - test.startTime)/1000 > 0 && test.status !== 'QUEUED'"
```
Comment (DmitryMishin) [inline client/app/containers/test-details/test-details.html]: You can add spaces before and after `/` ;)

File: client/app/containers/test-details/test-details.html [hunk 2]
```diff
@@ -137,17 +137,26 @@
...
+                                                      ng-class="test.status === 'QUEUED' ? 'grey_font' : 'light_text'"
+                                                      class="test-info__item">
                                                     <i class="material-icons">schedule</i>
                                                     <span>{{::(test.finishTime - test.startTime)/1000 | number:0}} seconds</span>
                                                 </span>
-                                                <span name="testOwner" data-ng-if="test.owner" data-ng-class="test.status === 'QUEUED' ? 'grey_font' : 'light_text'">
+                                                <span name="testOwner"
+                                                      ng-if="test.owner"
+                                                      class="test-info__item"
+                                                      ng-class="test.status === 'QUEUED' ? 'grey_font' : 'light_text'">
```
Comment (DmitryMishin) [inline client/app/containers/test-details/test-details.html]: codestyle. Please normalize tabs

File: client/app/containers/test-details/test-details.html [hunk 3]
```diff
@@ -137,17 +137,26 @@
...
                                                     <i class="fa fa-user" aria-hidden="true"></i>
-                                                    {{::test.owner}}
+                                                    <span>{{::test.owner}}</span>
                                                 </span>
-                                                <span name="device" data-ng-if="test.testConfig.device" data-ng-class="test.status === 'QUEUED' ? 'grey_font' : 'light_text'">
+                                                <span name="device"
+                                                      ng-if="test.testConfig.device"
+                                                      class="test-info__item"
+                                                      ng-class="test.status === 'QUEUED' ? 'grey_font' : 'light_text'">
```
Comment (DmitryMishin) [inline client/app/containers/test-details/test-details.html]: codestyle. Please normalize tabs

File: client/app/containers/test-details/test-details.html [hunk 4]
```diff
@@ -178,20 +187,28 @@
...
-                                                        <md-icon md-font-library="material-icons" ng-if="!artifact.extension" class="::material-icons">link</md-icon>
-                                                        <p>{{::artifact.name}}</p>
+                                                        <md-icon md-font-library="material-icons" ng-if="!artifact.extension" class="material-icons">link</md-icon>
+                                                        <span>{{::artifact.name}}</span>
                                                         <md-tooltip md-direction="bottom">{{::artifact.name}}</md-tooltip>
                                                     </a>
+                                                    <span name="testArtifactsBroken"
+                                                       aria-label="Test artifacts"
+                                                       class="test-settings__attachment _broken"
+                                                       ng-if="::artifact.hasBrokenLink">
```
Comment (DmitryMishin) [inline client/app/containers/test-details/test-details.html]: you use 3 spaces instead of 4

**Review by dzmitrymishyn — CHANGES_REQUESTED** [review]

File: client/app/containers/test-details/test-details.html (hunk 1 above)
Comment (danymill) [reply client/app/containers/test-details/test-details.html]: fixed

File: client/app/containers/test-details/test-details.html (hunk 2 above)
Comment (danymill) [reply client/app/containers/test-details/test-details.html]: fixed

File: client/app/containers/test-details/test-details.html (hunk 3 above)
Comment (danymill) [reply client/app/containers/test-details/test-details.html]: fixed

File: client/app/containers/test-details/test-details.html (hunk 4 above)
Comment (danymill) [reply client/app/containers/test-details/test-details.html]: fixed

**Review by dzmitrymishyn — APPROVED** [review]

**Review by dzmitrymishyn — APPROVED** [review]

**Pull request merged**